features/rni.feature

```diff
@@ -0,0 +1,187 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van de categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
+      | code | omschrijving                                      |
+      | 101  | Belastingdienst (inzake heffingen en toeslagen)   |
+      | 201  | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+
+
+    Scenario: RNI-deelnemer op geleverd gegeven
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                            | waarde                                      |
+      | voornamen (02.10)               | Peter                                       |
+      | geslachtsnaam (02.40)           | Jansen                                      |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
```

Comment:
dit element komt voor op zowel pl als op persoon niveau. Welke wordt hier bedoeld?

Reply:
het element kan in elke categorie voorkomen waar RNI de gegevens van mag aanleveren. Het element geldt in de categorie waar de gegeven stap betrekking op heeft.
In dit geval is de Gegeven stap "de persoon ... heeft de volgende gegevens", dus gaat het over persoon (lo3_pl_persoon met type='P')
In andere scenario's gaat het over "de persoon ... heeft de volgende 'inschrijving' gegevens" (rni-deelnemer in lo3_pl) of "de persoon ... heeft de volgende 'verblijfplaats' gegevens" (rni-deelnemer in lo3_pl_verblijfplaats), enz.